cranelift/filetests/filetests/isa/x64/imax.clif

```diff
@@ -0,0 +1,25 @@
+test compile precise-output
+target x86_64
+
+function %f0(i8, i8) -> i8 {
+block0(v0: i8, v1: i8):
+    v2 = imax.i8 v0, v1
+    return v2
+}
+
+; VCode_ShowWithRRU {{
+;   Entry block: 0
+; Block 0:
+;   (original IR block: block0)
+;   (instruction range: 0 .. 9)
+;   Inst 0:   pushq   %rbp
+;   Inst 1:   movq    %rsp, %rbp
+;   Inst 2:   cmpb    %dil, %sil
+;   Inst 3:   movl    %edi, %edi
```

Comment:
@Mrmaxmeier, thanks for the PR! I just wanted to point out here that this is another instance of #3744 (no action needed, just wanted to capture instances of that problem).